src/data/thunks.js

```diff
@@ -98,6 +97,32 @@ export function getProctoringSettings() {
   };
 }
 
+// todo: should this happen on its own or within updateAttemptAfter or somewhere else?
+// todo: need to update exam state to include exam access ticket for 1599 or later work?
+export function getExamAccessTicket() {
+  return async (dispatch, getState) => {
+    const { exam } = getState().examState;
+    if (!exam.id) {
+      logError('Failed to get exam settings. No exam id.');
+      handleAPIError(
+        { message: 'Failed to fetch exam access ticket. No exam id was found.' },
+        dispatch,
+      );
+      // return;
+    }
+    // try {
```

Comment:
Note: I commented this out because the commit was failing due to some missing parts, but this is the main chunk where I have questions about the implementation. A review of this currently commented block would be really helpful!